api/ssj/ssj.js

```diff
@@ -18,6 +18,15 @@ const api = axios.create({
   },
 });
 
+// have one function that gets a data structure used for entire dashboard
+
+async function dashboard() {
```

Comment:
i'm onboard to have just one call for the dashboard, to bootstrap the first page when they sign in